cmd/calyptia/create_core_instance_gcp.go

```diff
@@ -102,16 +105,19 @@ func newCmdCreateCoreInstanceOnGCP(config *config, client gcp.Client) *cobra.Com
 
 			cmd.Printf("[*] Waiting for create operation...")
 
+			limiter := rateLimiter.NewLimiter(rateLimiter.Every(1*time.Minute/rateLimit), 1)
```

Comment:
Make this a var/const.

Reply:
i didnt got, some specific value?

Reply:
RateLimit := rateLimiter.Every(time.Minute/rateLimit) ?